### ISSUE TITLE
`nvm install latest` results in `index out of range [3] with length 3`

### ISSUE THREAD

**Issue by SimonCockx** [coreybutler/nvm-windows#969]
**Issue:**
I wanted to install the new 20.0.0 NodeJS version by running `nvm install latest`. This resulted in the following error instead:
```
> nvm install latest
panic: runtime error: index out of range [3] with length 3

goroutine 1 [running]:
main.checkVersionExceedsLatest({0x11c973f0, 0x6})
        C:/Users/corey/OneDrive/Documents/workspace/oss/coreybutler/nvm-windows/src/nvm.go:790 +0x220
main.install({0x11c120b0, 0x6}, {0xb2fa65, 0x2})
        C:/Users/corey/OneDrive/Documents/workspace/oss/coreybutler/nvm-windows/src/nvm.go:302 +0x35d
main.main()
        C:/Users/corey/OneDrive/Documents/workspace/oss/coreybutler/nvm-windows/src/nvm.go:87 +0xa69
```

Running `nvm install 20.0.0` results in the same error.

**How To Reproduce:**
1. Run `nvm install latest` or `nvm install 20.0.0`
2. See error

**Expected Behavior:**
I expected the latest version of NodeJS to be installed.

**Desktop:**
 - OS: Windows 11

**Additional context:**

```
> nvm ls available

|   CURRENT    |     LTS      |  OLD STABLE  | OLD UNSTABLE |
|--------------|--------------|--------------|--------------|
|    20.0.0    |   18.16.0    |   0.12.18    |   0.11.16    |
|    19.9.0    |   18.15.0    |   0.12.17    |   0.11.15    |
|    19.8.1    |   18.14.2    |   0.12.16    |   0.11.14    |
|    19.8.0    |   18.14.1    |   0.12.15    |   0.11.13    |
|    19.7.0    |   18.14.0    |   0.12.14    |   0.11.12    |
|    19.6.1    |   18.13.0    |   0.12.13    |   0.11.11    |
|    19.6.0    |   18.12.1    |   0.12.12    |   0.11.10    |
|    19.5.0    |   18.12.0    |   0.12.11    |    0.11.9    |
|    19.4.0    |   16.20.0    |   0.12.10    |    0.11.8    |
|    19.3.0    |   16.19.1    |    0.12.9    |    0.11.7    |
|    19.2.0    |   16.19.0    |    0.12.8    |    0.11.6    |
|    19.1.0    |   16.18.1    |    0.12.7    |    0.11.5    |
|    19.0.1    |   16.18.0    |    0.12.6    |    0.11.4    |
|    19.0.0    |   16.17.1    |    0.12.5    |    0.11.3    |
|   18.11.0    |   16.17.0    |    0.12.4    |    0.11.2    |
|   18.10.0    |   16.16.0    |    0.12.3    |    0.11.1    |
|    18.9.1    |   16.15.1    |    0.12.2    |    0.11.0    |
|    18.9.0    |   16.15.0    |    0.12.1    |    0.9.12    |
|    18.8.0    |   16.14.2    |    0.12.0    |    0.9.11    |
|    18.7.0    |   16.14.1    |   0.10.48    |    0.9.10    |

This is a partial list. For a complete list, visit https://nodejs.org/en/download/releases
```

**Comment by ibotlold** [comment]
Had same issue, upgrade `nvm-windows` itself

**Comment by coreybutler** [comment]
Please search the prior issues before opening a new issue. This is the 9th duplicate of #955, and resolved in v1.1.11.

**Comment by jastorgalj** [comment]
En la ultima version funciona.
https://github.com/coreybutler/nvm-windows/releases
